bokeh/models/plots.py

```diff
@@ -350,11 +350,12 @@ def add_glyph(self, source_or_glyph: Glyph | ColumnarDataSource, glyph: Glyph |
         self.renderers.append(g)
         return g
 
-    def add_tile(self, tile_source: TileSource, **kwargs: Any) -> TileRenderer:
+    def add_tile(self, tile_source: Union[TileSource, xyzservices.TileProvider, str], retina=False, **kwargs: Any) -> TileRenderer:
         ''' Adds new ``TileRenderer`` into ``Plot.renderers``
 
         Args:
-            tile_source (TileSource) : a tile source instance which contain tileset configuration
+            tile_source (TileSource, xyzservices.TileProvider, str) : a tile source instance which contain tileset configuration
+            retina (bool) : use retina version of tiles (where available)
```

Comment:
```suggestion
            tile_source (TileSource, xyzservices.TileProvider, str) : 
                A tile source instance which contain tileset configuration
            
            retina (bool) : 
                Whether to use retina version of tiles (if available)
```
We are note completely consistent in adhering to this format yet, but it's what I'd like to have for docstrings throughout.